api/http/handler/endpoints/endpoint_status_inspect.go

```diff
@@ -14,13 +15,21 @@ type stackStatusResponse struct {
 	Version int
 }
 
+type edgeSchedule struct {
```

Comment:
Why are we mixing the `edgeSchedule` and `edgeJob` concepts? Is that any different?

Reply:
this is a decorated edgeJob, I'll rename it (it's basically the old edgeSchedule object that the agent expects)